crypto/rsa/rsa_pk1.c

```diff
@@ -270,6 +272,232 @@ int RSA_padding_check_PKCS1_type_2(unsigned char *to, int tlen,
     return constant_time_select_int(good, mlen, -1);
 }
 
+
+int ossl_rsa_prf(OSSL_LIB_CTX *ctx,
+                 unsigned char *to, int tlen,
+                 const char *label, int llen,
+                 const unsigned char *kdk,
+                 uint16_t bitlen)
+{
+    int pos;
+    int ret = 0;
```

Comment:
ret = -1 and remove all the ret = -1 below?

Reply:
true, it will make forgetting to set `ret = -1` on error less likely too, changed